packages/dd-trace/test/appsec/sdk/utils.spec.js

```diff
@@ -0,0 +1,115 @@
+'use strict'
+
+const { assert } = require('chai')
+
+const { getRootSpan } = require('../../../src/appsec/sdk/utils')
+const Tracer = require('../../../src/proxy')
+const Config = require('../../../src/config')
+
+describe('Appsec SDK utils', () => {
+  let tracer
+
+  before(() => {
+    tracer = new Tracer(new Config({
+      enabled: true
+    }))
+    tracer.init({})
```

Comment:
[constructor for tracer](https://github.com/DataDog/dd-trace-js/blob/master/packages/dd-trace/src/proxy.js#L37) (`packages/dd-trace/src/proxy.js`) expects no parameters.

Reply:
You removed `init` params but not `constructor`'s

Reply:
:no_good: